src/DependencyInjection/DI.External.Tests/test/Windsor.cs

```diff
@@ -0,0 +1,37 @@
+using Castle.Facilities.AspNetCore;
+using Castle.Windsor;
+using Microsoft.Extensions.DependencyInjection.Specification;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public class WindsorDependencyInjectionSpecificationTests : SkippableDependencyInjectionSpecificationTests
+    {
+        private static readonly WindsorContainer Container = new WindsorContainer();
+        private class HomeController
+        { }
+        public override string[] SkippedTests => throw new NotImplementedException();
```

Comment:
Throwing here is causing all the tests to fail. If you have no skipped tests you need to return an empty array:

```suggestion
        public override string[] SkippedTests => Array.Empty<string>();
```